docs/pages/guides/upgrading/0.19.mdx

```diff
@@ -0,0 +1,220 @@
+---
+meta:
+  title: "Upgrading to 0.19"
+  parentTitle: "Guides"
+  description: "Guide to upgrade to Liveblocks version 0.19"
+---
+
+```bash
+npm update @liveblocks/client@0.19 @liveblocks/react@0.19
+```
+
+In the Liveblocks 0.19 release, we’re adding support for Zustand v4 to
+`@liveblocks/zustand`. Zustand v4 brings greatly improved TypeScript types to
+its APIs, enabling us to enhance the quality of our types, bringing it in line
+with our React package.
+
+This release also brings several changes to `@liveblocks/react`, which improve
+rendering performance and stability. We have also refactored our internal
+packages to increase code sharing.
```

Comment:
```suggestion
This release also brings several improvements to `@liveblocks/react`, such as enhanced rendering performance and stability. Additionally, we have refactored our internal packages to facilitate code sharing.
```

The repetition of 'also' feels a bit redundant; my suggestion comes from Notion.ai, so feel free to change it